pkgs/applications/misc/hyperspace/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, fetchurl, lib
+, autoPatchelfHook, dpkg
+, libX11, libXext, libXi, libXau, libXrender, libXft, libXmu, libSM, libXcomposite, libXfixes, libXpm
+, libXinerama, libXdamage, libICE, libXtst, libXaw, fontconfig, pango, cairo, glib, libxml2, atk, gtk3
+, gdk-pixbuf
+# more
+, nss, nspr
+, libXScrnSaver
+, alsaLib
+# rt?
+, udev, libGL
```

Comment:
```suggestion
, gdk-pixbuf, nss, nspr, libXScrnSaver, alsaLib, udev, libGL
```

Reply:
d'oh sorry for leaving these in there! I do like grouping sometimes for maintainability but these are absolutely useless categories haha.  Anyway will fix.